services/rspamd/conf/worker-controller.inc

```diff
@@ -1,3 +1,7 @@
 bind_socket = "*:11334";
+{% if RSPAMD_PASSWORD %}
+password = "{{ RSPAMD_PASSWORD }}";
+{% else %}
 password = "mailu";
+{% endif %}
```

Comment:
This seems to define new functionality, which  does not necessarily belong to Kubernetes. A change like this should be proposed in a separate pull request. Futhermore, I beleive (assumption) that the password is passed to the rspamd page by Admin. Changing overriding the password will break this behavior.

Reply:
OK I can separate those

Reply:
@kaiyou Is this true that the password is provided by Admin and not always `mailu`
- If the password is always `mailu` I will create another PR providing this functionality
- If the password is provided by Admin, any hints on how to configure are welcome and I will remove the lines above